packages/interface/src/components/fullscreen-mode/style.scss

```diff
@@ -10,14 +9,80 @@ body.js.is-fullscreen-mode {
 			height: calc(100% + #{ $admin-bar-height });
 		}
 
-		#adminmenumain,
+		// Hide the top adminbar toolbar.
 		#wpadminbar {
 			display: none;
 		}
 
+		// Display the sidebar offscreen, and then transition in when needed.
+		#adminmenumain {
+			position: relative;
+			top: 60px;
+			left: -160px;
+			transition: left 0.12s ease-in-out;
+		}
+
+		// Hides the admin menu
+		// This prevents keyboard navigation from accessing inner links
+		#adminmenuwrap {
+			display: none;
+		}
+
+		#adminmenu {
+			position: relative;
+			bottom: auto;
+		}
+
+		#adminmenuback {
+			top: 60px;
+		}
+
+		.edit-post-header-toolbar__block-toolbar,
+		.block-editor-editor-skeleton__body,
+		.block-editor-editor-skeleton__footer {
+			transition: margin-left 0.12s ease-in-out;
+		}
+
 		#wpcontent,
 		#wpfooter {
 			margin-left: 0;
 		}
 	}
 }
+
+body.js.is-showing-admin-menu {
+	#adminmenumain {
+		left: 0;
+	}
+
+	// Show the admin menu
+	#adminmenuwrap {
+		display: block;
+	}
+
+	.edit-post-header-toolbar__block-toolbar,
+	.block-editor-editor-skeleton__body,
```

Comment:
I renamed this to `InterfaceSkeleton` in https://github.com/WordPress/gutenberg/pull/21335, so the classnames here will need to be updated.

Reply:
Thanks for the heads up. I've updated the classnames.